modules/programs/gnome-terminal.nix

```diff
@@ -30,10 +30,41 @@ let
           description = "The bold color, null to use same as foreground.";
         };
 
+        allowBold = mkOption {
+          default = null;
+          type = types.nullOr types.bool;
+          description = "If true, allow applications in the terminal to make text boldface.";
+        };
+
         palette = mkOption {
           type = types.listOf types.str;
           description = "The terminal palette.";
         };
+
+        cursorForegroundColor = mkOption {
```

Comment:
The `Color` in the end can be omitted from this and the below options.

I think the naming of `foregroundColor`, `backgroundColor`, and `boldColor` was a mistake and they should have been named without the `Color`. I might change this in the future.